Zend/zend_vm_def.h

```diff
@@ -1049,14 +1050,13 @@ ZEND_VM_C_LABEL(assign_op_object):
 				break;
 			}
 		}
-		cache_slot = (OP2_TYPE == IS_CONST) ? CACHE_ADDR((opline+1)->extended_value) : NULL;
+		cache_slot = (OP2_TYPE == IS_CONST) ? CACHE_ADDR((opline+1)->extended_value) : _cache_slot;
```

Comment:
How does this change the cost for non CONST op2?
On one hand now ``get_property_ptr_ptr`` has to make more work, on the other hand ``zend_object_fetch_property_type_info`` is eliminated.
Can you measure the difference, please.

Reply:
With this micro bench under callgrind:

``` php
<?php

class C {
    public int $a = 0;
}

$obj = new C();
$a = 'a';
for ($i = 0; $i < 1_000_000; $i++) {
    $obj->$a += 2;
}
```

```
base:  mean:  264465884.0000;  stddev:  0.0000;  diff:  +0.00%
pr:    mean:  250464574.0000;  stddev:  0.0000;  diff:  -5.29%
alt:   mean:  246467362.0000;  stddev:  0.0000;  diff:  -6.81%
```

`pr` is this branch, and [`alt`](https://github.com/php/php-src/compare/master...arnaud-lb:php-src:oss-fuzz-71407-alt) is an alternative in which I added a `zend_property_info **prop_info_p` parameter to `zend_object_read_property_t` and `zend_object_get_property_ptr_ptr_t`.

`alt` is faster than `pr` for op2 != CONST, however it's slower for op2 = CONST:

``` php
<?php

class C {
    public int $a = 0;
}

$obj = new C();
for ($i = 0; $i < 1_000_000; $i++) {
    $obj->a += 2;
}
```

```
base:  mean:  192463206.0000;  stddev:  0.0000;  diff:  +0.00%
pr:    mean:  192461896.0000;  stddev:  0.0000;  diff:  -0.00%
alt:   mean:  196464687.0000;  stddev:  0.0000;  diff:  +2.08%
```